slack.ts

```diff
@@ -0,0 +1,54 @@
+/** Error20:
+    - line 21: don't know how to declare call signature in tsc*/
+
+import * as querystring from "querystring";
+import fetch from "node-fetch";
+
+const tkn = process.env.SLACK_TOKEN;
+const domain = process.env.DOMAIN;
+
+async function sendMessage(message : string, config : string) : Promise<void> {
```

Comment:
Can we have the config be something like:
```typescript
interface Config {
  channels: string[];
}
```
e.g. so we can support sending a message to multiple channels?